src/winetricks

```diff
@@ -16029,6 +16029,85 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata origin apps \
+    title="EA Origin" \
+    publisher="EA" \
+    year="2011" \
+    media="download" \
+    file1="OriginSetup.exe" \
+    installed_file1="${W_PROGRAMS_X86_WIN}/Origin/Origin.exe" \
+    homepage="https://www.origin.com/"
+
+helper_origin_dl()
+{
+    # Skipping checksum as this changes too often
+    w_download_to origin https://origin-a.akamaihd.net/Origin-Client-Download/origin/live/OriginSetup.exe
+}
+
+load_origin()
+{
+    # Need to force wine-6.0 as Origin doesn't run below WineCX21 (wine32on64)
+    if [ "$(uname -s)" = "Darwin" ] && w_wine_version_in ,6.0 ; then
+        w_die "${W_PACKAGE} requires wine version 6.0 (or newer)"
+    fi
+
+    if [ "${WINETRICKS_FORCE}" = 0 ] && w_workaround_wine_bug 44691 "Installer fails under wine, manually unpacking it instead" 6.7,; then
+        w_call originupdater
+        w_warn "${W_PACKAGE} might fail, to update use the originupdate verb"
+    else
+        helper_origin_dl
+        w_try_cd "${W_CACHE}/${W_PACKAGE}"
+        w_try "${WINE}" "${file1}" /NoLaunch ${W_OPT_UNATTENDED:+/SILENT}
+    fi
+
+    if w_workaround_wine_bug 32342 "QtWebEngineProcess.exe crashes when updating or launching Origin (missing fonts)"; then
+        w_call corefonts
+    fi
+
+    if w_workaround_wine_bug 44985 "Disabling libglesv2 to make Store and Library function correctly."; then
+        w_override_dlls disabled libglesv2
+    fi
+
+    # Origin requirements
+    w_call vcrun2010
+    w_call vcrun2015
+    w_call vcrun2019
```

Comment:
These should be shrunk down to only `vcrun2019` & `d3dcompiler_47` the other two were added due to OriginSetup also installing those during setup.

Reply:
Not sure why this doesn’t show outdated…

Removed vcrun2015 since vcrun2019 already overs this is forgotten….